src/main/java/net/neoforged/neoforge/common/Tags.java

```diff
@@ -134,15 +134,18 @@ public static class Blocks {
          */
         public static final TagKey<Block> ORE_BEARING_GROUND_STONE = tag("ore_bearing_ground/stone");
         /**
-         * Ores which on average result in more than one resource worth of materials
+         * Ores which on average result in more than one resource worth of materials ignoring fortune and other modifiers.
+         * (example, Redstone Ore)
          */
         public static final TagKey<Block> ORE_RATES_DENSE = tag("ore_rates/dense");
         /**
-         * Ores which on average result in one resource worth of materials
+         * Ores which on average result in one resource worth of materials ignoring fortune and other modifiers.
+         * (Example, Coal Ore)
```

Comment:
Prime example that needs to be listed here is metal ores like iron or gold. Use overworld gold if you want to contrast with the sparse nether gold ore